### ISSUE TITLE
Classification describing why datasets are private

### ISSUE THREAD

**Issue by askarlupka** [EDRN/labcas-ui#153]
## 🤔 Tell Us About the Feature

Currently there are 23 private datasets. They are potentially private for different reasons: study is incomplete, biospecimen repository needs to be completely used up, etc. If I were a researcher looking for datasets, I would want to know if there were private datasets that were currently going through cleaning/processing and may be publicly available later. But as of right now it's those are not distinguishable from permanently private datasets. Thus, it would be useful to know just the reason behind why they are private. 

## 🎇 What Solution You'd Like

Could put something on the data dashboard that visualizes this. Each of the datasets would need a characteristic describing why it is private. Ideally something concise and descriptive.

**Comment by hoodriverheather** [comment]
@askarlupka good idea. We will look at the best way to track/indicate this in LabCAS.
